documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -2474,7 +2484,7 @@ The following table lists relevant properties for configuring parallel execution
 | ```junit.jupiter.execution.parallel.config.fixed.max-pool-size```
 | Desired maximum pool size of the underlying fork-join pool for the ```fixed```
   configuration strategy
-| a positive integer, must greater than or equal to `junit.jupiter.execution.parallel.config.fixed.parallelism`
+| a positive integer, must be greater than or equal to `junit.jupiter.execution.parallel.config.fixed.parallelism`
 | 256 + the value of `junit.jupiter.execution.parallel.config.fixed.parallelism`
 
 | ```junit.jupiter.execution.parallel.config.fixed.saturate```
```

Comment:
We should add an entry for the `dynamic` one as well.

Reply:
That one is was `junit.jupiter.execution.parallel.config.dynamic.factor` and was already documented here: https://github.com/junit-team/junit5/blame/07f2bf349e45f5a63228cb6002ae7bdbd3445ce1/documentation/src/docs/asciidoc/user-guide/writing-tests.adoc#L2463-L2467

The full set is:
```
junit.jupiter.execution.parallel.config.dynamic.factor
junit.jupiter.execution.parallel.config.dynamic.max-pool-size-factor

junit.jupiter.execution.parallel.config.fixed.parallelism
junit.jupiter.execution.parallel.config.fixed.max-pool-size
```

Reply:
https://github.com/junit-team/junit5/pull/3206/commits/6849f00963c4f33c991cfd2b9dd3009ebab461d9